meta/main.yml

```
@@ -9,6 +9,9 @@ galaxy_info:
   license: "license (BSD, MIT)"
   min_ansible_version: 2.8
   platforms:
+    - name: Arch Linux
```

Should be `ArchLinux`:

```suggestion
    - name: ArchLinux
```